index.html

```diff
@@ -14,6 +15,13 @@
     <link rel="manifest" href="src/img/manifest.json">
     <meta name="msapplication-config" content="src/img/browserconfig.xml">
     <meta name="theme-color" content="#3a84d0">
+    <!-- Twitter -->
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:title" content="Alicante Frontend">
+    <meta name="twitter:description" content="El ecosistema Frontend es un caos y ¡nos encanta! Si quieres aprender y compartir sobre desarrollo web orientado al Frontend has llegado al sitio correcto.">
+    <meta name="twitter:site" content="@AlicanteFront">
+    <meta name="twitter:creator" content="@AlicanteFront">
+    <meta name="twitter:image:src" content="/src/img/alicante-frontend-logo.svg">
```

Comment:
Sadly SVG is not supported for the `twitter:image` meta tag. See https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary

![twitter-image-metatag-no-svg](https://user-images.githubusercontent.com/7225802/31117524-72a5dfce-a82a-11e7-96f3-b7102456c79f.png)

You'll need to convert the logo to PNG or JPG and make different sizes depending on Facebook or Twitter. For Facebook it's recommended 1200x630 px and for Twitter 1024x512 px ([source](https://www.rakacreative.com/blog/social-media/social-media-image-sizes-always-up-to-date/)).

Also, remove the initial slash in the of `/src/img/alicante-frontend-logo.svg`.